themes/digital.gov/src/scss/new/_authors-list.scss

```diff
@@ -45,6 +45,9 @@
         @include u-margin(0);
         @include u-font("sans", "3xs");
       }
+      h5 {
+        margin: 0;
+      }
     }
```

Comment:
Let's target classes for styling instead of HTML elements.

Reply:
Style removed to be address in other PR